app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Api::CommentsController < ApplicationController
+  skip_before_action :authenticate_user!
+  def index
+    @comments = Comment.all
+    render json: @comments
+  end
+
+  def create
+    @comment = Comment.new(
+      text: comment_params[:text],
+      users_id: comment_params[:users_id]
```

Comment:
- [ ] According to your schema file there is no attribute named `users_id`. But you are trying to create a comment with an attribute named `users_id`, this will probably raise **UnknowAttributeError**. May be you meant `user_id`, kindly check it.

- [ ] The `post_id` is also missing from your code to create a new comment. Kindly check it.